README.md

```diff
@@ -8,9 +8,26 @@ A Jest reporter that creates compatible junit xml files
 yarn add --dev jest-junit
 ```
 
+## Important Notice
+In an upcoming major version 5.x jest-junit will no longer function as a testResultProcessor. It will only work as a jest reporter. See the docs just below this for how to transition your project.
+
 ## Usage
 In your jest config add the following entry:
 ```JSON
+{
+  "reporters": [ "jest-junit" ]
```

Comment:
we should keep the default reporters, right? cause this will overwrite them.
to keep them we should specify it as :
```json
reproters: ["default", "jest-junit"]
```

Reply:
Good call. Amending now.